### PR TITLE
Added more tests & fixed mock responses

### PR DESCRIPTION
**Pull request by rogercyyu**
- Fixed mock responses
- Added some more tests

### CODE REVIEW

File: tests/test_checkFile.py [hunk 1]
```diff
@@ -27,38 +27,98 @@ def test_no_file_exception():
...
+    link = "http://google.com"
+    cF = checkFile.checkFile(args)
+    cF.headRequest(link)
+    cF.printAll()
+    captured = capsys.readouterr()
+    assert captured.out == "\x1b[91m[400] http://google.com\x1b[0m\n"
+
+
+@mock.patch("urllib3.PoolManager.request", side_effect=mocked_requests_get)
+def test_printBadResultsJSON(self, capsys):
```
Comment (TDDR) [inline tests/test_checkFile.py]: Could you change the title to better represent what's being tests
`test_printUnknownResult`

File: tests/test_checkFile.py [hunk 2]
```diff
@@ -27,38 +27,98 @@ def test_no_file_exception():
...
+    link = "thisdoesnotwork"
+    cF = checkFile.checkFile(args)
+    cF.headRequest(link)
+    cF.printAll()
+    captured = capsys.readouterr()
+    assert captured.out == "\x1b[90m[???] thisdoesnotwork\x1b[0m\n"
+
+
+@mock.patch("urllib3.PoolManager.request", side_effect=mocked_requests_get)
+def test_checkThatFile(self, capsys):
```
Comment (TDDR) [inline tests/test_checkFile.py]: Same here. Could you change the title to better represent what's being tests
test_printAllJSON

**Review by TDDR — COMMENTED** [review]
Looks good, just a couple of minor changes to the function names at it's good to merge!

**Comment by rogercyyu** [comment]
Okay, changes are addressed. Thanks for the feedback.

**Review by TDDR — APPROVED** [review]

**Pull request merged**